package.json

```diff
@@ -141,6 +147,9 @@
     "typescript": "3.5.3",
     "webpack": "4.39.2"
   },
+  "bin": {
+    "instantsearch-scripts": "./scripts/index.js"
```

Comment:
we need to make sure these are published on npm via the `files` key

Reply:
`./scripts/index.js` is not inside `files` key...?